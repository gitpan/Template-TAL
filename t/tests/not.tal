<html xmlns:tal="http://xml.zope.org/namespaces/tal">

  <div tal:define="foo string:0" tal:condition="not:foo">0</div>
  <div tal:define="foo string:1" tal:condition="not:foo">1</div>
  
</html>
