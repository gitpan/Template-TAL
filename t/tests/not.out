<html>

  <div>0</div>

</html>
